hightouch_blueprints/execute_sync.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+import errors
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--sync-id', dest='sync_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def execute_sync(sync_id, access_token):
+    """
+    Executes/starts a Hightouch Sync
+    see: https://hightouch.io/docs/api-reference/#operation/TriggerRun
+    """
+    
+    sync_api = f"https://api.hightouch.io/api/v1/syncs/{sync_id}/trigger"
+    api_headers = {
+        'authorization': f"Bearer {access_token}",
+        'Content-Type': 'application/json'
+    }
+    payload = {
+        "fullResync": "false"
+    }
```

Comment:
Since this is a valid thing to want as true, we should make it an optional argument.